### PR TITLE
feature: orders & fixes

### PR DESCRIPTION
**Pull request by SergeyMarynenko**
- Fix navbar
- Add orders functional

### CODE REVIEW

File: Modules/Translation/Resources/lang/core/en/core.php [hunk 1]
```diff
@@ -81,5 +81,7 @@
...
+    'youPhoto' => 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+    'ordersTitle' => 'Orders',
+    'statusUpdate' => 'Update status of order is success'
```
Comment (7s4r) [inline Modules/Translation/Resources/lang/core/en/core.php]: Add a trailing comma at the end

File: Modules/Translation/Resources/lang/core/fr/core.php [hunk 2]
```diff
@@ -81,5 +81,7 @@
...
+    'youPhoto' => 'Le Lorem Ipsum est simplement du faux texte employé dans la composition et la mise en page avant impression. Le Lorem Ipsum est le faux texte standard de l\'imprimerie depuis les années 1500, quand un peintre anonyme assembla ensemble des morceaux de texte pour réaliser un livre spécimen de polices de texte. Il n\'a pas fait que survivre cinq siècles, mais s\'est aussi adapté à la bureautique informatique, sans que son contenu n\'en soit modifié. Il a été popularisé dans les années 1960 grâce à la vente de feuilles Letraset contenant des passages du Lorem Ipsum, et, plus récemment, par son inclusion dans des applications de mise en page de texte, comme Aldus PageMaker.',
+    'ordersTitle' => 'Ordres',
+    'statusUpdate' => 'L\'état de la mise à jour de l\'ordre est un succès'
```
Comment (7s4r) [inline Modules/Translation/Resources/lang/core/fr/core.php]: Add a trailing comma at the end

File: Modules/User/Http/Controllers/AuthController.php [hunk 3]
```diff
@@ -44,7 +45,7 @@ public function callback()
 
         $credentials = [
             'email' => $authUser->email,
-            'password' => bcrypt($authUser->facebook_pw)
+            'password' => $authUser->facebook_pw
```
Comment (7s4r) [inline Modules/User/Http/Controllers/AuthController.php]: Add a trailing comma at the end

File: Modules/User/Http/Controllers/AuthController.php [hunk 4]
```diff
@@ -379,9 +381,17 @@ public function postBuyBitcoins(Request $request)
 
             if (count($res['error']) > 0) {
                 flash()->error("Error with payment on kraken");
+
                 return redirect()->back()->withInput();
             } else {
+                Order::create([
+                    'user_id' => $this->auth->id(),
+                    'bitcoins' => $buyPrice,
+                    'wallet' => $request->bitcoinWallet
```
Comment (7s4r) [inline Modules/User/Http/Controllers/AuthController.php]: Add a trailing comma at the end

File: composer.json [hunk 5]
```diff
@@ -57,14 +57,10 @@
   "scripts": {
     "post-install-cmd": [
       "php artisan clear-compiled",
-      "php artisan stylist:publish",
-      "php artisan module:publish",
```
Comment (7s4r) [inline composer.json]: why you removed this?

File: database/migrations/2014_07_02_230147_migration_cartalyst_sentinel.php [hunk 6]
```diff
@@ -101,12 +101,27 @@ public function up()
...
+
+        Schema::create ('orders', function (Blueprint $table)
+        {
+            $table->increments('id');
+            $table->integer('user_id')->unsigned();
+            $table->double('bitcoins');
+            $table->string('status')->default('process');
+            $table->string('wallet');
+            $table->timestamps();
+
```
Comment (7s4r) [inline database/migrations/2014_07_02_230147_migration_cartalyst_sentinel.php]: Useless line

File: public/themes/flatly/css/style.css [hunk 7]
```diff
@@ -1142,12 +1142,17 @@ div.panel-payment label > span {
     display: none;
 }
 
+
```
Comment (7s4r) [inline public/themes/flatly/css/style.css]: Useless line

File: public/themes/flatly/css/style.css [hunk 8]
```diff
@@ -1162,4 +1167,8 @@ div.panel-payment label > span {
     .exmenu{
         border: none;
     }
+
+    .top-nav-collapse {
+        padding: 0;
+    }
 }
```
Comment (7s4r) [inline public/themes/flatly/css/style.css]: EOF

File: resources/views/asgard/user/public/account.blade.php [hunk 9]
```diff
@@ -140,6 +140,49 @@
                         </div>
                         <div id="menu2" class="tab-pane fade">
                             <h3>Order history</h3>
```
Comment (7s4r) [inline resources/views/asgard/user/public/account.blade.php]: Translation

File: composer.json (hunk 5 above)
Comment (SergeyMarynenko) [inline composer.json]: Its need for CMS. CMS generated new files. Its not need for us

**Pull request merged**